src/components/Tr/Tr.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+function Tr(props) {
+
+    return (
+        <tr>
+            <td>{props.index + 1}</td>
+            <td>{props.obj.orderName}</td>
+            <td>{props.obj.orderPhone}</td>
+            <td>{props.obj.orderFast ? 'YES': 'NO'}</td>
+            <td>{props.obj.orderPrice}</td>
+            <td>{Object.entries(props.obj.orderProducts).map((obj, index, arr) => {
+                    if (index !== arr.length - 1) {
+                        return `${obj[0]}: ${obj[1]}, `
+                    } else {
+                        return `${obj[0]}: ${obj[1]}`
```

Comment:
o_O

same as 14 line

what's the point to have this `if...else` statement then?

Reply:
they aren`t same. The last element should be without comma in the end